README.md

```diff
@@ -367,3 +367,31 @@ Promise.resolve("Hello!")
 ```
 
 If you don't `catch` a rejected promise, it will throw an error and crash (the same as if you didn't catch an exception in a try/catch statement).
+
+# Persisting the state
+
+Let's make sure that changes in our list are persisted between apps restarts. To do that, we will be using [`AsyncStorage`](https://facebook.github.io/react-native/docs/asyncstorage.html).
+
+To retrieve something stored, we can use `getItem`: it accepts a key and it returns a promise, the result will be either the previously stored data, or `null` if there isn't any.
+
+To retrieve the state when the application is initialized, we can use one of the lifecycle callbacks of a component (they're present only in class components), [`componentDidMount`](https://reactjs.org/docs/react-component.html#componentdidmount) and set the retrieved state inside the promise callback.
```

Comment:
Are the class components explained before this doc?

Reply:
Yep, in the bit about altering the state (few chapters before)